libs/core-ui/src/lib/util/TelemetryEventName.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export enum TelemetryEventName {
+  // Error analysis
+  ErrorAnalysisTreeMapTabClick = "ErrorAnalysisTreeMapTabClick",
+  ErrorAnalysisTreeMapFeatureListClick = "ErrorAnalysisTreeMapFeatureListClick",
+  ErrorAnalysisTreeMapClearSelection = "ErrorAnalysisTreeMapClearSelection",
+  ErrorAnalysisTreeMapSaveAsNewCohortClick = "ErrorAnalysisTreeMapSaveAsNewCohortClick",
+  ErrorAnalysisTreeMapCohortSaved = "ErrorAnalysisTreeMapCohortSaved",
+  ErrorAnalysisHeatMapTabClick = "ErrorAnalysisHeatMapTabClick",
+  ErrorAnalysisHeatMapQuantileBinningClick = "ErrorAnalysisHeatMapQuantileBinningClick",
+  ErrorAnalysisNewMetricSelected = "ErrorAnalysisNewMetricSelected",
+  // Main menu
+  MainMenuCohortSettingsClick = "MainMenuCohortSettingsClick",
+  MainMenuDashboardConfigurationClick = "MainMenuDashboardConfigurationClick",
+  MainMenuSwitchCohortClick = "MainMenuSwitchCohortClick",
+  MainMenuNewCohortClick = "MainMenuNewCohortClick",
+  // Cohort
+  NewCohortAdded = "NewCohortAdded"
+  // Counterfactual
```

Comment:
button click events for counterfactual and other components will coming in the next PR